.changeset/config.json

```diff
@@ -9,7 +9,7 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": ["@blockprotocol/site", "@local/*"],
+  "ignore": ["@blockprotocol/site", "@crates/*", "@local/*"],
```

Comment:
Without root `tsconfig.json`, the only way for us to lint `crates/type-system/scripts/build-wasm.ts` is to create a new workspace at `crates/type-system`. We don't want to publish it, hence this change in Changeset config